test/controller/spec/base-controller.spec.js

```diff
@@ -517,6 +517,43 @@ describe('Form Controller', () => {
       form.validate.should.have.been.calledOn(form);
     });
 
+    describe('sanitise inputs', () => {
+      const tests = [
+        { value: 'HELLO\/*TEST*\/WORLD1', expected: 'HELLOTESTWORLD1' },
+        { value: 'HELLO|WORLD2', expected: 'HELLOWORLD2' },
+        { value: 'HELLO&&WORLD3', expected: 'HELLO&WORLD3' },
+        { value: 'HELLO@@WORLD4', expected: 'HELLO@WORLD4' },
+        { value: 'HELLO/..;/WORLD5', expected: 'HELLOWORLD5' },
+        { value: 'HELLO......WORLD6', expected: 'HELLO.WORLD6' },
+        { value: 'HELLO/eTc/paSsWdWORLD7', expected: 'HELLOWORLD7' },
+        { value: 'HELLOC:\\WORLD8', expected: 'HELLOWORLD8' },
+        { value: 'HELLOcMd.ExEWORLD9', expected: 'HELLOWORLD9' },
+        { value: 'HELLO<WORLD10', expected: 'HELLO<-WORLD10' },
+        { value: 'HELLO>WORLD11', expected: 'HELLO>-WORLD11' },
+        { value: 'HELLO[WORLD12', expected: 'HELLO[-WORLD12' },
+        { value: 'HELLO]WORLD13', expected: 'HELLO]-WORLD13' },
+        { value: 'HELLO~WORLD14', expected: 'HELLO~-WORLD14' },
+        { value: 'HELLO&#WORLD15', expected: 'HELLO&#-WORLD15' },
+        { value: 'HELLO%UWORLD16', expected: 'HELLO%U-WORLD16' },
+        {
+          value: '1/*2*/3|4&&5@@6..7/etc/PASSwd8C:\\9Cmd.eXe10/..;/11<12>13[14]15~16&#17%U18',
+          expected: '1234&5@6.7891011<-12>-13[-14]-15~-16&#-17%U-18'
+        }
+      ];
+
+      tests.forEach(({value, expected}) => {
+        it('sanitisation returns correct data', function () {
```

Comment:
cool. Do we think it wise to have some tests for standard input to ensure we aren't accidentally changing inputs that normally wouldn't be changed?

Reply:
Happy paths will be tested as part of the existing acceptance tests so it might be duplication, but I'm happy to add a couple for completeness.